src/custom-webpack/dev-server/schema.d.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * Created by Evgeny Barabanov on 01/07/2018.
+ */
+
+import {BuildWebpackServerSchema} from '@angular-devkit/build-angular/src/server/schema';
+
+export interface CustomWebpackDevServerBuildSchema extends BuildWebpackServerSchema {
```

Comment:
Should extend DevServerBuilderOptions

Reply:
Same here, should be `BrowserBuilderSchema`